packages/xrpl/src/sugar/autofill.ts

```diff
@@ -57,6 +57,50 @@ async function autofill<T extends Transaction>(
   return Promise.all(promises).then(() => tx)
 }
 
+/**
+ * Autofills fields in a transaction. This will set `Sequence`, `Fee`,
+ * `lastLedgerSequence` according to the current state of the server this Client
+ * is connected to. It also converts all X-Addresses to classic addresses and
+ * flags interfaces into numbers.
+ *
+ * @param this - A client.
+ * @param transactions - A {@link Transaction} in JSON format
+ * Only used for multisigned transactions.
+ * @returns The autofilled transaction.
+ */
+async function autofillBatch<T extends Transaction>(
+  this: Client,
+  transactions: Array<{ transaction: T; signersCount?: number }>,
+): Promise<T[]> {
+  // Sender Account to Sequence map
+  const map: Map<string, number> = new Map()
```

Comment:
```suggestion
  const accountSequenceMap: Map<string, number> = new Map()
```
I think a descriptive name is actually useful here.